drivers/include/pca9633.h

```diff
@@ -0,0 +1,523 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_pca9633 PCA9633 I2C PWM controller
+ * @ingroup     drivers_actuators
+ * @brief       Device driver for the NXP PCA9633
+ *
+ * @{
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ * @file
+ */
+
+#ifndef PCA9633_H
+#define PCA9633_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "stdbool.h"
+#include "periph/i2c.h"
+
+// Register definitions (page 11, table 7)
+
+/**
+ * @brief Mode register 1
+ */
+#define PCA9633_REG_MODE1       0x00
+
+/**
+ * @brief Mode register 2
+ */
+#define PCA9633_REG_MODE2       0x01
+
+/**
+ * @brief Brightness control LED0
+ */
+#define PCA9633_REG_PWM0        0x02
+
+/**
```

Comment:
Move all those register defines to `drivers/pca9633/include/pca9633_registers.h`, no need to have them in the global API space.

Reply:
E.g. `PCA9633_REG_PWM0` to `PCA9633_REG_PWM3` are needed when calling `pca9633_set_pwm` (see `tests/driver_pca9633/main.c` on line 90). Should they still be declared in `pca9633_regs.h` or in `pca9633.h`? It's my first driver in RIOT ;)

Reply:
Usually it's a bit odd for an API to let the user write registers directly, but for writing PWM channels that's simple enough.

You might add some type-safety (and ease discoverability) by making the register definitions an `enum` instead of `#defines`

```C
typedef enum {
    PCA9633_REG_PWM0 = 0x2,
    PCA9633_REG_PWM1 = 0x3,
    …
} pca9633_reg_t;
```

The you would have

```C
void pca9633_set_pwm(pca9633_t* dev, pca9633_reg_t reg_pwm, uint8_t pwm)
```

Although it would make for an even easier to use API if `pca9633_set_pwm()` would simply take channel 0…3 as parameter.

Unfortunately we don't have a common API for external PWM devices yet, but it would still be a good advice to follow the API in `drivers/include/periph/pwm.h`.